### PR TITLE
Add documents about SSO. Bump up version to 1.0.0.

### PR DESCRIPTION
**Pull request by bochunz**
@wora PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,3 +141,31 @@ file `~/.oauth2l.token`.
 ```
 $ oauth2l reset
 ```
+
+## Options
+
+### `--json`
+
+Use a JSON key file downloaded from
```
Comment (wora) [inline README.md]: Specifies an OAuth credential file, either OAuth client ID or Service Account key, to start the OAuth flow. You can download the file from ...

Please link to the credential page, not the main page.

File: README.md [hunk 2]
```diff
@@ -141,3 +141,31 @@ file `~/.oauth2l.token`.
...
+[Google Cloud Console](https://console.cloud.google.com/).
+
+```
+$ oauth2l fetch --json ~/service_account.json cloud-platform
+```
+
+### `--sso` and `--sso_cli`
+
+Use an SSO CLI to fetch OAuth token. The SSO CLI outputs an
```
Comment (wora) [inline README.md]: an single sign-on (SSO) command to .... The command outputs an OAuth access token to its stdout. The default command is for Google's corporate SSO.

File: README.md (hunk 1 above)
Comment (bochunz) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (bochunz) [inline README.md]: Done

**Comment by bochunz** [comment]
Will change the default token to bare in a following commit.

File: README.md [hunk 3]
```diff
@@ -141,3 +141,34 @@ file `~/.oauth2l.token`.
...
+```
+$ oauth2l fetch --json ~/service_account.json cloud-platform
+```
+
+### `--sso` and `--sso_cli`
+
+A single sign-on (SSO) command to fetch OAuth token. 
+The command outputs an OAuth access token to its stdout. 
+The default command is for Google's corporate SSO.
+It works like:
```
Comment (wora) [inline README.md]: Change "It" to "The command"

**Pull request merged**